cmd/cloudshell_open/appfile.go

```diff
@@ -32,9 +32,14 @@ type env struct {
 	Required    *bool  `json:"required"`
 }
 
+type scripts struct {
+	Postdeploy string `json:"postdeploy"`
```

Comment:
I like this structure.

I'll now think if it makes sense if each member of `scripts` should be a list (`[]string`) so people can specify multiple script files.